src/geovista/geodesic.py

```diff
@@ -279,20 +279,20 @@ def mesh(self) -> pv.PolyData:
 
         Examples
         --------
-        Add a bounding-box to the plotter for the ``africa`` panel of a cubed-sphere.
-        The geodesic bounding-box is generated from the 4 corners of the cubed-sphere
-        panel located over Africa. Natural Earth coastlines are also rendered along
+        Add a ``C32`` bounding-box around North Africa and Western Europe.
+        The geodesic bounding-box is generated by defining 4 longitude-latitude
+        corners. Natural Earth coastlines are also rendered along
         with a texture mapped Natural Earth base layer.
 
         >>> import geovista
-        >>> from geovista.geodesic import panel
+        >>> from geovista.geodesic import BBox
         >>> plotter = geovista.GeoPlotter()
         >>> _ = plotter.add_base_layer(
         ...     texture=geovista.natural_earth_hypsometric(), style="wireframe"
         ... )
-        >>> bbox = panel("africa", c=32)
+        >>> bbox = BBox(lons=[-20, 20, 20, -20], lats=[10, 10, 50, 50], c=32)
         >>> _ = plotter.add_mesh(bbox.mesh, color="white")
-        >>> plotter.view_yz()
+        >>> plotter.camera.zoom(1.3)
```

Comment:
Keeping the default angle and adjusting the zoom provides a clearer view of how `BBox` exists both under and above the user's mesh.